s2wav/s2let_math.py

```diff
@@ -0,0 +1,83 @@
+"""Compute basic math tiling. """
+import numpy as np
+
+
+def f_s2dw(k, B):
```

Comment:
Definitely don't follow the bad practices from `s2let`!  This give functions more useful names, e.g. `tiling_integrand` perhaps in this case?